templates/conf/kubernetes.conf.erb

```diff
@@ -258,6 +258,9 @@
 <filter kubernetes.**>
   @type kubernetes_metadata
   @id filter_kube_metadata
+  kubernetes_url "#{ENV['KUBERNETES_URL'] || https://#{ENV['KUBERNETES_SERVICE_HOST']}:#{ENV['KUBERNETES_SERVICE_PORT']}/api}"
```

Comment:
"KUBERNETES_" prefix seems general.
How about "FLUENTD_FILTER_KUBERNETS_URL" or something?
Other environment variable too.

Reply:
For `KUBERNETES_SERVICE_HOST` and `KUBERNETES_SERVICE_PORT` this is referring to the standard variables that kubernetes makes available inside each pod but I guess you know that already.
I put them here to match the default https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter/blob/master/lib/fluent/plugin/filter_kubernetes_metadata.rb#L195 as we're specifying `kubernetes_url` here in any case.

I changed `KUBERNETES_URL` to `FLUENT_FILTER_KUBERNETS_URL`, rebased from master re-ran `make src-all`.